### DISCUSSION TITLE
Can't open steam save

### DISCUSSION THREAD

**Discussion by Sors01** [zencq/NomNom#168] [Q&A]
I played the game through Game Pass and Nomnom easily opened and edited the save. Then I decided to play on the steam version, I started a new game so that the save appeared. But I can't open it, the Nomnom freezes immediately after i push open button, and I also can't transfer the save from the game pass to steam in any way
![PIC](https://github.com/zencq/NomNom/assets/165648407/7ed9b6ef-6df4-4651-b331-4ea3534a5969)

**Comment by Flipgirl** [comment]
same here, I updated the app and every time I try to transfer the safe file from XB to ST it freezes and does nothing :( help! lol

**Comment by flanp** [comment]
@Sors01 You need to open the game on Steam and create a save. After that, it will generate the folder so it can be detected by nomnom.
